maven-release-plugin/src/site/apt/examples/perform-release.apt.vm

```diff
@@ -29,7 +29,7 @@ Perform a Release
 
   Performing a release runs the following release phases {{{../../maven-release-manager/#perform}by default}}:
 
-   * Checkout from an SCM URL with optional tag
+   * Checkout from an SCM URL with optional tag to <<<workingDirectory>>> (<<<target/checkout>>> by default)
```

Comment:
I'd prefer to use the official property instead of default `target`.

Reply:
it is documentation, not code: in code, the official property is used
in documentation, I wrote the default value that is the concrete value that people know

Reply:
Alright, makes sense.